README.md

```diff
@@ -1 +1,133 @@
-# Hello-react-frontend
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [Live Demo](#live-demo)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+# 📰 Hello-rails-frontend <a name="about-project"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://react.dev/">react</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://github.com/Divinity-dev/Hello-rails-backend">react app</a>
+
```

Comment:
- [ ] [OPTIONAL] It would be nice if you change the text of `"react app"` to `"RoR"` 👍🏼